### ISSUE TITLE
[build] Create release version container images

### ISSUE THREAD

**Issue by stp-ip** [txtdirect/txtdirect#203]
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What happened**:
We currently provide commit based container images.

**What you expected to happen**:
Provide per release images such as `c.txtdirect.org/txtdirect:v0.4.0` on tags.

**Comment by erbesharat** [comment]
I read cloud builder documentation but couldn't find any way to define a dynamic value for a substitute variable.
For example, what I had in my mind was:
```
substitutions:
  _TAG: dev-${SHORT_SHA}

images:
  - "gcr.io/$PROJECT_ID/txtdirect:${_TAG}"
```
Then I wanted to change ${_TAG}'s value inside the GCB trigger (Release trigger) but it's not possible. It can't get the value from ${SHORT_SHA} and put it inside ${_TAG}. It generates this command to build the image:
`build -t gcr.io/erfanb-223517/txtdirect:dev-${SHORT_SHA} .`

One thing that we can do is to add an additonal step to run a bash script that writes the tag or commit hash inside a file and then read that file to build the image.
Something like this:
```
- name: 'ubuntu'
  args: ['bash', '-c', 'GET_TAG_OR_COMMIT > _TAG']

- name: gcr.io/cloud-builders/docker
entrypoint: sh
args: ['-c', 'docker build -t gcr.io/$PROJECT_ID/txtdirect:$(cat _TAG) .']
```

I don't really like this solution. One of the down sides from GCP docs:

> The difference between using the `images` field and the Docker push command is that if you use the `images` field, the stored image will be displayed in the build results. This includes the Build description page for a build in the GCP Console, the results of Build.get(), and the results of gcloud builds list. However, if you use the Docker push command to store the built image, the image will not be displayed in the build results.

**Comment by stp-ip** [comment]
Was looking into this also. Will see, if I can get some insights from the GCB team at the next conference, but currently it's either scripting around or doing it manually. I think doing it manually with release documentation is the better way. We don't release that often and I'm ok to do the extra step for now.

Does that sound reasonable. Aka revisit later?

**Comment by erbesharat** [comment]
This issue gets fixed in #298 using https://github.com/txtdirect/txtdirect/pull/298/files#diff-1c9505c2db3442eaad5d37b81c28aa3a.
The build triggers in GCP are also ready.

**Comment by stp-ip** [comment]
Saw that, but wasn't sure it would already be triggered automatically once we merge. Awesome. Thought it might still need a few changes on GCB.
